functionaltests/src/test/java/util/WebDriverSofortueberweisung.java

```diff
@@ -31,7 +31,7 @@
     private final String tan;
 
     public WebDriverSofortueberweisung(final String pin, final String tan) {
-        super(BrowserVersion.FIREFOX_38, true);
+        super(BrowserVersion.FIREFOX_45, true);
```

Comment:
45 is the latest available FF version in the latest Selenium version. I'd like to update Selenium major version to 3, but since it is a significant update - better to do this as a separate issue/PR.